backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -56,22 +56,16 @@ public Paged<User> Get(UserFilter filter = null)
         {
             this.User.ThrowIfNotAuthorized(Permissions.AdminUsers);
 
-            var userAgencies = User.GetAgencies();
-            if (userAgencies == null || !userAgencies.Any())
-            {
-                throw new NotAuthorizedException("Current user does not belong to an agency");
-            }
-
             var query = this.Context.Users
                 .Include(u => u.Agencies)
                 .ThenInclude(a => a.Agency)
```

Comment:
I think this will remain an `inner join` which will result in not returning users who are not part of an `agency` or `role`.

Reply:
I am seeing users with no agency or role. I want to test a theory in dev. Testing how having both sys-admin and agency-admin affects the results. After my test, I will another PR with the query change.